README.md

```diff
@@ -1,221 +1,172 @@
+## How to use the Frank!Flow
 
-The easiest way to get started with the Frank!Flow is with help of the [Frank!Runner](https://github.com/ibissource/frank-runner). You can skip this if you already have a Frank!Runner.
+### Frank!Runner
 
-### Frank2Frank!Flow
+The easiest way to get started with the Frank!Flow is with help of the [Frank!Runner](https://github.com/ibissource/frank-runner). Startup the [Frank2Example4](https://github.com/ibissource/frank-runner#frank2example4) and go to `localhost/frank-flow` in your browser.
 
-Create a new frank folder next to the frank-runner folder. Following the name convention this should be called `frank2frank-flow`. This folder will contain three main files and an optional file to restart the runner.
+### Maven enabled Frank
 
-#### POM
+You can add the Frank!Flow to an existing Frank by adding it to the POM.
 
-The POM will tell the Frank!Runner which extra dependencies you want to download. In this case we want the artifact “frank-flow”. Create a file called `pom.xml` and place the following text in it:
+The POM will tell Maven which extra dependencies you want to download. In this case we want the artifact “frank-flow”. Add the following dependencies and repository to the `pom.xml`:
 
 ```XML
-<project
-    xmlns="http://maven.apache.org/POM/4.0.0"
-    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd"
-    >
-    <modelVersion>4.0.0</modelVersion>
-
-    <groupId>org.ibissource</groupId>
-    <artifactId>frank2frank-flow</artifactId>
-    <version>1.0</version>
-    <packaging>war</packaging>
-
-    <dependencies>
-        <dependency>
-            <groupId>org.ibissource</groupId>
-            <artifactId>ibis-adapterframework-webapp</artifactId>
-            <version>[7.6,)</version>
-            <type>war</type>
-        </dependency>
-        <dependency>
-            <groupId>org.ibissource</groupId>
-            <artifactId>frank-flow</artifactId>
-            <version>0.0.1-SNAPSHOT</version>
-        </dependency>
-    </dependencies>
-
-    <repositories>
-        <repository>
-            <id>ibissource</id>
-            <name>Ibissource</name>
-            <url>https://nexus.ibissource.org/content/repositories/public</url>
-        </repository>
-    </repositories>
-
-</project>
+<dependencies>
+    <dependency>
+        <groupId>org.ibissource</groupId>
+        <artifactId>ibis-adapterframework-webapp</artifactId>
+        <version>[7.7,)</version>
+        <type>war</type>
+    </dependency>
+    <dependency>
+        <groupId>org.ibissource</groupId>
+        <artifactId>frank-flow</artifactId>
+        <version>[2.2,)</version>
+    </dependency>
+</dependencies>
+
+<repositories>
+    <repository>
+        <id>ibissource</id>
+        <name>Ibissource</name>
+        <url>https://nexus.ibissource.org/content/repositories/public</url>
+    </repository>
+</repositories>
 ```
```

Comment:
I have the following remarks here:
- I did not test that these Maven coordinates are correct. Did you do that?
- I do not like that the reader needs our Nexus repository. Can the Frank!Flow be released on Maven Central?

Reply:
It is copied from the Frank that's used in the cypress tests.

It does work but to put it exactly in the right spot can be tricky for beginners, but they should use the runner anyway.

It could be usefull to link to the POM of that Frank in the repo as an example.

Reply:
Ok for now.